sdk/core/azure-core/test/e2e/azure_hang.cpp

```diff
@@ -1,74 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// SPDX-License-Identifier: MIT
-
-/**
- * @brief Performs  upload and download for n times trying to catch long running bugs.
- * Originally set to use 50Mb for 2000 times, which takes ~2.5hours to complete.
- *
- */
-
-#if defined(_MSC_VER)
-// this option is used to allow the application to use std::getenv without getting a compilation
-// warning about it on MSVC.
-#define _CRT_SECURE_NO_WARNINGS
-#endif
-
-#include <iostream>
-
-#include <azure/storage/blobs/blob.hpp>
-
-#define BLOB_SIZE 50 * 1024ULL * 1024
-#define REPEAT_FOR 2000
-#define CONCURRENCY 16
-
-int main()
-{
-  using namespace Azure::Storage::Blobs;
-
-  std::string containerName = "sample-container";
-  std::string blobName = "sample-blob";
-  std::string blobContent;
-  blobContent.resize(BLOB_SIZE, 'c');
-
-  std::string connString(std::getenv("STORAGE_CONNECTION_STRING"));
-
-  auto containerClient = BlobContainerClient::CreateFromConnectionString(connString, containerName);
```

Comment:
This is an important test to keep running as a unit test, at least as part of live tests. @Jinming-Hu, @katmsft - do we have such a test case already?